packages/quantic/force-app/main/default/lwc/quanticSort/__tests__/quanticSort.test.js

```diff
@@ -145,14 +195,158 @@ describe('c-quantic-sort', () => {
 
   afterEach(() => {
     cleanup();
+    searchStatusState = defaultSearchStatusState;
+  });
+
+  describe('when an initialization error occurs', () => {
+    beforeEach(() => {
+      mockErroneousHeadlessInitialization();
+    });
+
+    afterAll(() => {
+      mockSuccessfulHeadlessInitialization();
+    });
```

Comment:
You shouldn't need to do that, since your higher `beforeEach` will already `mockSuccessfulHeadlessInitialization()`

Reply:
That's true indeed 👍

Reply:
You should be able to remove that `afterAll` entirely @lvu285